main.scss

```diff
@@ -8,10 +8,60 @@
 @import 'src/scss/mixins';
 @import 'src/scss/variables';
 
+/// Outputs all features and styles available for o-footer-services.
+/// @param {map} $opts [('logo': 'ftlogo-v1:origami', 'icons': ('slack', 'github'))] - A map of optional footer features to output, including the footer logo and list of social icon links.
+/// @param {bool} $include-base-styles [true] - Whether to output fundamental o-footer-services styles required by the optional features.
+/// @access public
+@mixin oFooterServices($opts: (
+	'logo': false,
+	'icons': ('slack', 'github')
+), $include-base-styles: true) {
+	$logo: map-get($opts, 'logo');
+	$icons: map-get($opts, 'icons');
 
-@if $o-footer-services-is-silent == false {
-	@include oFooterServicesBase();
+	// Output fundamental o-footer-services styles required by all users.
+	@if($include-base-styles) {
+		@include _oFooterServicesBase();
+	}
+
+	// Output classes for a logo in the footer.
+	@if($logo) {
+		@include _oFooterServicesLogo($logo);
+	}
+
+	// Output classes for footer icons.
+	@if($icons) {
+		.o-footer-services__icon-link {
+			@include oTypographyBold('sans');
+			@include _oFooterServicesLink(
+				$custom-link-base: _oFooterServicesGet('default-link-color'),
+				$custom-link-hover: _oFooterServicesGet('default-link-hover'),
+				$custom-link-background: _oFooterServicesGet('default-background')
+			);
+
+			display: inline-block;
+			min-width: max-content;
+			margin: 0 1em 1.5em 0;
+		}
 
+		// Output a footer class for each requested icon.
+		@each $icon in $icons {
+			// Allowed social images: https://registry.origami.ft.com/components/social-images
+			$allowed-social-images: (facebook, github, linkedin, pinterest, slack, tumblr, twitter, whatsapp, youtube);
```

Comment:
Should we add something to the social images documentation which indicates that this may need updating if new images are added?

Reply:
It would feel like an odd place to document that. As they don't change often, I wonder if the error message should just point to us instead? 'Contact Origami if you would like $icon to be available '

Reply:
yeah that could work 👍